terraform/app_service.tf

```diff
@@ -40,8 +40,32 @@ resource "azurerm_linux_web_app" "main" {
     }
   }
 
+  sticky_settings {
+    app_setting_names = [
+      "APPINSIGHTS_INSTRUMENTATIONKEY",
+      "APPINSIGHTS_PROFILERFEATURE_VERSION",
+      "APPINSIGHTS_SNAPSHOTFEATURE_VERSION",
+      "APPLICATIONINSIGHTS_CONFIGURATION_CONTENT",
+      "APPLICATIONINSIGHTS_CONNECTION_STRING",
+      "ApplicationInsightsAgent_EXTENSION_VERSION",
+      "DJANGO_ALLOWED_HOSTS",
+      "DJANGO_OAUTH_AUTHORITY",
+      "DJANGO_OAUTH_CLIENT_ID",
+      "DJANGO_SECRET_KEY",
+      "DJANGO_TRUSTED_ORIGINS",
```

Comment:
From [`linux_web_app#sticky_settings`](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/linux_web_app#sticky_settings):

> [app_setting_names](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/linux_web_app#app_setting_names) - (Optional) A list of app_setting names that the Linux Web App will not swap between Slots when a swap operation is triggered.

I think most of these `DJANGO_*` settings should _not_ be sticky, they correspond to the slots.

On this list, the only one that may be the same across slots is `DJANGO_OAUTH_CLIENT_ID` (but even that could change once we finish with #621).

Not on this list, but could be added (they are not currently slot settings):

* `DJANGO_INIT_PATH` - now that we have the mounts normalized for each env, this is the same)
* `DJANGO_OAUTH_CLIENT_NAME` - just the internal app identifier for the OAuth client